lib/companies/jobs.ex

```diff
@@ -86,8 +86,10 @@ defmodule Companies.Jobs do
       {:error, %Ecto.Changeset{}}
 
   """
-  def delete_job(%Job{} = job) do
-    Repo.delete(job)
+  def delete_job(%Job{} = job, user) do
+    job
+    |> Job.changeset(%{})
```

Comment:
We shouldn't need to create a changeset just to delete.

Reply:
This is one of the things i was working on today, but didn't finish until i left home. The whole thing needs some rework, plus there are some holes we have in the original branch as well. I am going to work on them tomorrow and during the weekend.